CASMINST-5582 Update Management_Node_Image_Customization.md (#2787)
Cray-HPE/docs-csm: operations/configuration_management/Management_Node_Image_Customization.md

1. (`ncn-mw#`) Update boot parameters for a Kubernetes NCN.

>    **NOTE:** If you are following this procedure as part of the multiproduct recipe upgrade, you should only modify NCN worker boot parameters. Master and storage NCNs are managed automatically in later stages of the CSM upgrade.

Reviewer: ```suggestion
   **NOTE:** If you are following this procedure as part of the multi-product recipe upgrade, you should only modify NCN worker boot parameters.
   Master and storage NCNs are managed automatically in later stages of the CSM upgrade.
```

>    

Reviewer: ```suggestion

```

>    Master and storage NCNs are managed automatically in later stages of the CSM upgrade.

Reviewer: ```suggestion
   Master and storage NCNs are managed automatically in later stages of the CSM upgrade.

```
